crates/node/ops/mod.rs

```diff
@@ -7,8 +7,12 @@ pub mod http2;
 pub mod idna;
 pub mod ipc;
 pub mod os;
+pub mod process;
 pub mod require;
 pub mod util;
 pub mod v8;
+pub mod vm;
+mod vm_internal;
 pub mod winerror;
+pub mod worker_threads;
```

Comment:
This should probably staty disabled as well.

Reply:
Perfect